### DISCUSSION TITLE
Does npmjs.com delete very old blank packages/namespaces?

### DISCUSSION THREAD

**Discussion by ColinCyk** [npm/feedback#683] [Registry] [answered]
As per the title, I cannot find anywhere that npmjs.com publicly state 'we never delete packages/namespaces' - does this occur to prevent namespace squatting or similar?

**Comment by MylesBorins** [comment, marked as answer]
@ColinCyk we do not automatically delete any packages. Some package are able to be unpublished by the original author according to [our unpublish policies](https://docs.npmjs.com/policies/unpublish).

Occasionally packages / orgs / usernames are given to those requesting them via [our dispute resolution process](https://docs.npmjs.com/policies/disputes).
